packages/@lwc/template-compiler/src/parser/attribute.ts

```diff
@@ -32,7 +32,6 @@ import {
     ID_REFERENCING_ATTRIBUTES_SET,
     KNOWN_HTML_ELEMENTS,
     TEMPLATE_DIRECTIVES,
-    PROPS_ATTRS_TRANSFORMS,
```

Comment:
You can now get rid of the `PROPS_ATTRS_TRANSFORMS` object itself in the `constants.ts` as `propertyToAttributeName` was the last consumer.

Reply:
Great catch! Updated.